src/style.css

```
@@ -1 +1,113 @@
-/* Styles go here */
+body {
+  margin: 0;
+  
+}
+
+* {
+  box-sizing: border-box;
+
+}
+
+.head {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: auto 0;
+
+}
```

```suggestion
  margin: auto 0;
}
```

ці пусті строки непотрібні, їх краще видатити